docs/packaging/index.md

```diff
@@ -43,6 +43,25 @@ Additonally, we need a few more tools to carry out the packaging process:
 sudo eopkg it git arcanist solbuild solbuild-config-unstable
 ```
 
+## Setting up a Dev Tracker Account
+
+The [Solus Dev Tracker](https://dev.getsol.us) is where all packaging patches are submitted and reviewed. It runs [Phabricator](https://www.phacility.com/phabricator/), and we use `arcanist` to submit patches directly from the command line.
+
+To submit patches to  you will need an acount. [Register a new account](https://dev.getsol.us/auth/start/) with your email address, or log in with a Github account. 
```

Comment:
Remove double space

```suggestion
To submit patches to you will need an acount. [Register a new account](https://dev.getsol.us/auth/start/) with your email address, or log in with a Github account. 
```

Reply:
"To submit patches to you will need an acount."
submit patches to what? Missing a word? This could just say 
"To submit patches you will need an account"

Also, 'acount' is misspelled.

Otherwise, looks good.

Reply:
Brutal. Good catch. I'll fix this up in a new commit